action-templates/actions/action-maven-release/action.yml

```diff
@@ -55,10 +55,10 @@ runs:
         cache: "maven"
         cache-dependency-path: "./${{ inputs.app-path}}/pom.xml"
         server-id: "central"
-        server-username: ${{ inputs.CENTRAL_USERNAME }}
-        server-password: ${{ inputs.CENTRAL_PASSWORD }}
+        server-username: CENTRAL_USERNAME
+        server-password: CENTRAL_PASSWORD
```

Comment:
aber hat die action überhaupt darauf dann Zugriff

Reply:
die werden im nächsten Step nochmal explizit gesetzt: https://github.com/it-at-m/lhm_actions/pull/89/files#diff-d2bbc590c62811ce2cd3902bf3557e406ac39c947967e6698808247622391ae8R83-R86